### PR TITLE
Update 2016-04-05-newmarket.md

### PR DESCRIPTION
**Pull request by ldunh**
Would like another screen shot.

### CODE REVIEW

**Comment by dtzar** [comment]
Looks good to me.

File: _posts/2016-04-05-newmarket.md [hunk 1]
```diff
@@ -72,6 +72,8 @@ By enabling this build definition, the bottleneck for waiting for manual builds
 
 Next, the automated tests that were typically run by QA later in the process were introduced as Visual Studio Test tasks in automated builds. The shorter tests (such as unit tests and a smaller list of feature tests) were added into the continuous integration build, and the longer tests were added into a scheduled nightly build. Previously, a manual process by quality assurance was involved to categorize tests, but NewMarket could categorize in test classes and test methods and include that as part of the build definition and quality of the build. 
 
+UPDATE: add screenshot
```
Comment (dtzar) [inline _posts/2016-04-05-newmarket.md]: Yes, I'd really like to have a screenshot of the test automation filters used with VSTS.

Comment (ldunh) [inline _posts/2016-04-05-newmarket.md]: Is this where you want to add it?

Comment (ldunh) [inline _posts/2016-04-05-newmarket.md]: Add it here?

**Pull request merged**